pyproject.toml

```
@@ -62,6 +62,12 @@ skip = "pp* cp38-*_aarch64 cp38-musllinux_*"
 # don't bother unless someone asks
 archs = ["native"]
 
+test-requires = [
+    "pytest",
+    "pytest-cov",
+]
+test-command = "pytest {project}/nitime/tests"
```

We aren't using `pytest-cov`, so I wouldn't bother. And we can use `--pyargs` to test with the installed version:

```suggestion
    "nitime[full]",  # Enable all optional behavior
]
test-command = "pytest -rsx --pyargs nitime"
```

---

> pytest-cov

I took this part from `requirements-dev.txt`, I'll remove it there, too

---

Ahh it's not used for cibuildwheel but *is* used in CIs, so I'll need to put it back in `requirements-dev.txt`.